### DISCUSSION TITLE
Barcode/QR Field Not Scanning PDF417 Codes

### DISCUSSION THREAD

**Discussion by AlexandriaLittle** [Budibase/budibase#9527] [Help]
Hello,

I'm struggling to get the new Barcode/QR Code field to scan the PDF417 barcode on the back of a Michigan Driver's License. My use case is integrating a sex offender database API with Budibase to search for possible offenders who may attempt to enter my company's building. The barcode field on the ID contains the person's first name, last name, and date of birth among other bits of information.

The issue that I have is that no matter how many different devices/cameras I try it seems like none of them are even recognizing that there's a barcode in the frame. I see PDF417 as a supported barcode but nothing seems to work. Could anyone offer guidance on what I could to to fix or could this be a bug?

**Comment by melohagan** [comment]
Hey @AlexandriaLittle 

There are a couple of open issues on the underlying library that suggest there possibly is an issue: https://github.com/zxing-js/library/issues?q=is%3Aissue+is%3Aopen+pdf417

**Reply by AlexandriaLittle** [reply]
Okay, I'll hope for it being fixed in the future then! Thank you!
